samples/LargeResponseApp/LargeResponseApp.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web" ToolsVersion="15.0">
+  <Import Project="..\..\build\common.props" />
```

Comment:
They don't. Do you recommend keeping it here or bringing in the relevant stuff from `common.props`?